src/processing.cpp

```diff
@@ -3,8 +3,10 @@
 #include "hooks.h"
 #include "loader.h"
 #include "unreal.h"
+#include "logger.h"
 
 using namespace ohl::unreal;
+using namespace ohl::logger;
```

Comment:
This using ends up redundant cause you always fully qualify the name. Perhaps try `using ohl::logger:log;` in the logger header, then just call `log` directly everywhere? Not sure if that clobbers an existing function.